Source/WSTagsField.swift

```diff
@@ -697,7 +700,9 @@ extension WSTagsField {
             oldIntrinsicContentHeight = newIntrinsicContentHeight
         }
 
-        self.isScrollEnabled = contentRect.height + contentInset.top + contentInset.bottom >= newIntrinsicContentHeight
+        if self.enableScrolling {        
```

Comment:
I think it's confusing for the lib user to have access to 2 different properties (`enableScrolling ` and `isScrollEnabled `) with the same goal. But makes sense what you did. I need to think about it.

Reply:
@ricardopereira That's a fair point! What can the user of the library do with `isScrollEnabled` right now? It seems like even if it's `false` (which is the default), the line I wrapped in an `if` will set it to true depending on the size of the tags field. Is it useful just to set scrolling enabled before the tag field grows big enough to do it automatically, or is there another use for `isScrollEnabled`?